README.md

```diff
@@ -108,14 +239,41 @@ If the argument you provide when calling the function holds values [a,b,c...z],
 Make sure to handle two special cases: length 0 (then return _true_) and length 1 (then return that single value).
 Again, don't use &&.
 
+Solution:
+
+```
+  function andN(values){
+  for (i = 0; i <= values.length; i++){
+    if (values[i] === false) {
+      return false;
+    } };
+    if (i = values.length + 2) {
+      return true;
+    }
+    ;}
```

Comment:
4c) You never actually get to return true in your loop, because i never reaches values.length + 2. Also, be sure to use `==` instead of `=` when checking for equality. And don't forget to test your code to see if it works!